samples/ubuntu-dual-stack.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: virt.virtink.smartx.com/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: ubuntu-dual-stack
+spec:
+  instance:
+    memory:
+      size: 1Gi
+    disks:
+      - name: ubuntu
+      - name: cloud-init
+    interfaces:
+      - name: pod
+  volumes:
+    - name: ubuntu
+      containerDisk:
+        image: smartxworks/virtink-container-disk-ubuntu
+    - name: cloud-init
+      cloudInit:
+        userData: |-
+          #cloud-config
+          password: password
+          chpasswd: { expire: False }
+          ssh_pwauth: True
+          runcmd:
+            - dhclient -6 ens4
+        # TODO: use networkData to configure DHCPv6, but currently only works for DHCPv4
```

Comment:
Is this a limitation of cloud-init?

Reply:
I'm not sure, but still trying to figure it out.